tools/internal_ci/linux/aws/grpc_aws_experiment_remote.sh

```diff
@@ -21,6 +21,9 @@ sudo apt install -y build-essential autoconf libtool pkg-config cmake python pyt
 sudo pip install six
 
 cd grpc
+
+# without port server running, many tests will fail
+python tools/run_tests/start_port_server.py
```

Comment:
This ssh session issue occurs only when starting the port server, correct? Perhaps the simpler solution is to ensure it shuts down?

Reply:
I think in general it can happen in many situations and we should not make assumptions about the properties of the test framework. in my experience, `bazel test` works fine and it doesn't prevent the ssh session from terminating, but there are other tests (for other gRPC languages) that use the `tools/run_tests/run_tests.py` wrapper and this wrapper does trigger the problem as well. Whether that's because of port_server.py (which is started by run_tests.py internally) or for some other reason I'm not sure (and the framework is complicated enough to make finding that out difficult). 
I like the solution I'm proposing because it will work in all situations and it doesn't make assumptions about the test framework / build tool. The hanging ssh session is hard thing to debug and it can create a lot of confusion so I think a general solution is appropriate even at the cost of adding a bit more complexity to the script.